holoviews/plotting/mpl/raster.py

```diff
@@ -126,6 +135,9 @@ def update_handles(self, key, axis, element, ranges, style):
 
 class QuadMeshPlot(ColorbarPlot):
 
+    nodata = param.Integer(default=None, doc="""
+        Missing data (NaN) value for integer data""")
```

Comment:
```suggestion
        Optional missing-data value for integer data.
        If non-None, data with this value will be replaced with NaN so that it is transparent when plotted.""")
```